nixos/modules/services/misc/gitolite.nix

```diff
@@ -49,6 +62,17 @@ in
         '';
       };
 
+      extraGitoliteRc = mkOption {
+        type = types.str;
```

Comment:
If it makes sense that extraGitoliteRc can be set in multiple modules, better change from types.str to types.lines so that multiple definitision can be merged.

Reply:
Well, I think it's possible for it to make sense. If anything, it can make implementation of the upcoming `enableGitAnnex` option a bit shorter. However, I can't think why **end users** would want to separate `services.gitolite` configuration into multiple modules.

Taking this in consideration, what would you advise? Should I change the type?

Reply:
If it works with types.lines I think we should do it. You never know when someone would like to modularize their configuration. (For instance, I have my samba smb.conf defined in two modules.)

Reply:
Ok, I'll make the change.